documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -226,30 +226,30 @@ apiVersion: forklift.konveyor.io/v1beta1
 kind: Plan
 metadata:
   name: <plan> <1>
-  namespace: {namespace}
+  namespace: <namespace>
 spec:
   warm: true <2>
   provider:
     source:
       name: <source_provider>
-      namespace: {namespace}
+      namespace: <namespace>
     destination:
       name: <destination_cluster>
-      namespace: {namespace}
+      namespace: <namespace>
   map:
     network: <3>
       name: <network_map> <4>
-      namespace: {namespace}
+      namespace: <namespace>
     storage:
       name: <storage_map> <5>
-      namespace: {namespace}
-  targetNamespace: {namespace}
+      namespace: <namespace>
+  targetNamespace: <namespace>
```

Comment:
replacing {namespace} with <namespace> here even more significant but we can take it a step further and highlight that it's usually supposed to be a different namespace
```suggestion
  targetNamespace: <target_namespace>
```